molgenis-core-ui/src/main/java/org/molgenis/ui/EntityListenerRepositoryDecorator.java

```diff
@@ -151,6 +170,22 @@ public void update(Iterable<? extends Entity> records)
 	}
 
 	@Override
+	public void update(Stream<? extends Entity> entities)
+	{
+		if (entityListeners != null)
+		{
+			entities = entities.filter(entity -> {
```

Comment:
forEach is a terminal operation, the decorated repository requires a stream that can be consumed.